src/dotnet/Chat.UI.Blazor/Pages/ChatPage.razor

```diff
@@ -14,8 +14,22 @@
     // Log.LogDebug("Rendered for #{ChatId}", chat.Id);
 }
 
+@if (ReferenceEquals(chat, SpecialChat.Unavailable) && m.IsInaccessiblePlace) {
+     <RenderIntoSlot Name="MainLeftDrawer">
+         <LeftPanel IsSkeletonWidePanel="@true">
+             <WidePanelBottomContent>
+                 <InaccessiblePlaceLeftPanelBottom/>
+             </WidePanelBottomContent>
+         </LeftPanel>
+     </RenderIntoSlot>
+}
+
 <RenderIntoSlot Name="MainHeader">
-    <ChatHeader @key="@chatKey" ChatContext="@chatContext" />
+    @if (ReferenceEquals(chat, SpecialChat.Unavailable)) {
```

Comment:
I don't like these conditions. I think it would be better if we pass state via ChatContext and then apply appropriate rendering insine ChatHeader component.